docs/application/dotnet/guides/nui/xaml/TypeConverter.md

```diff
@@ -0,0 +1,66 @@
+# TypeConverters and XAML
+
+This topic introduces the purpose of type conversion from string as a general XAML language feature. Such as converter "Red" to <code>Color.Red</code>. If you write a custom class, and you want instances of your class to be usable as XAML settable attribute values, you might need to write a custom TypeConverter class.
```

Comment:
To avoid plagiarism, please provide a link to Microsoft documentation (https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/typeconverters-and-xaml).

Copyright: https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx